### PR TITLE
Chapter 3: Static Pages

### PR DESCRIPTION
**Pull request by balqis277**
Ready

### CODE REVIEW

File: test/controllers/static_pages_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/static_pages_controller_test.rb:1]: You don't need to do test please delete this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
please fix my comment

**Pull request closed without merging**